README.md

```diff
@@ -1,2 +1,28 @@
 # Portfolio-Website
 Personal Portfolio website as a microverse day3 project
+
+># Project Details
+>this project focuses on building a personal portfolio website
+which will showcase the projects you have worked on as a developer, your personal achievements so far and why you think you fits in as a software developer in the tech industry.
+this is a complete website that microverse students are undertaken to enhance thier coding skills. 
+
+
+# Installation process(Packages)
+first add the .github folder followed by the steps below
+*npm install --save-dev hint@7.x
+*Copy .hintrc to the root directory of your project.
+*run npx hint .
+*npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+# Languages used
+*Html
+*Css
+
+👤 **Author**
+Sagoe Bright Marcus
+
+# Social Media Links
+
+- GitHub: [@github](https://www.linkedin.com/in/bright-marcus-sagoe-74335820a/)
+- Twitter: [@facebook](https://web.facebook.com/?_rdc=1&_rdr)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] Please use the Microverse ReadMe template headers as they are, they have proven effective and professional and are the standard for all Micronauts. In order to add steps to clone the repo and install packages, you can use the Getting Started header.